src/hooks/usePersonalData/localStorageClient/localStorageClient.tsx

```diff
@@ -121,3 +111,58 @@ const localStorageClient: PersonalDataClient = {
 }
 
 export default localStorageClient
+
```

Comment:
Any specific reason to move the types here? I think it was easier to read when they were in a separate file

Reply:
This is a workaround, the app won't start otherwise, see rocket chat for the full error logs

Reply:
Seems renaming the file from types.d.ts to types.ts will do the trick. Let's do that